README.md

```diff
@@ -6,81 +6,70 @@
-        |- random_circuits
-        |- randomized_benchmarking
-        |- utils
-    | mitiq_pyquil
-        |- conversions
-    	|- pyquil_utils
-        |- quil
-    | mitiq_qiskit
-    	|- conversions
-    	|- qiskit_utils
-    | zne
-        |- zne
-        |- inference
-        |- scaling
+```python
+import mitiq
+mitiq.about()
 ```
-## Installation
 
-To install locally use:
+This prints out version information about core requirements and optional quantum software packages which Mitiq can
+interface with. 
 
-```bash
-pip install -e .
-```
+### Supported quantum programming languages
 
-To install the requirements for development use:
+Mitiq can currently interface with
 
-```bash
-pip install -r requirements.txt
-```
+* [Cirq](https://github.com/quantumlib/Cirq) >= 0.9.0, 
+* [Qiskit](https://qiskit.org/) >= 0.19.0, and 
+* [pyQuil](https://github.com/rigetti/pyquil) >= 2.18.0. 
+
+Cirq is a core requirement of Mitiq and is automatically installed. To use Mitiq with other quantum programming
+languages, install the optional package(s) following the instructions linked above.
+
+### Supported quantum processors
+
+Mitiq can be used on any quantum processor which can be accessed by supported quantum programming languages and is 
+available to the user.
 
-Note that this will install our testing environment that depends
-on `qiskit` and `pyquil`.
+## Getting started
 
-## Use
-A [Getting Started](https://mitiq.readthedocs.io/en/latest/guide/guide_02-getting-started.html)
-tutorial can be found in the [documentation](https://mitiq.readthedocs.io).
+See this [getting started](https://mitiq.readthedocs.io/en/latest/guide/guide-getting-started.html) guide in 
+[Mitiq's documentation](https://mitiq.readthedocs.io).
 
+## Error mitigation techniques
 
-## Development and Testing
+Mitiq currently implements [zero-noise extrapolation](https://mitiq.readthedocs.io/en/latest/guide/guide-zne.html) and 
+is designed to support [additional techniques](https://github.com/unitaryfund/mitiq/wiki).
 
-Ensure that you have installed the development environment. Then you can run
-the tests using `make test` and build the docs using `make docs`. For more
-information, see the contributor's guide (linked below).
+## Documentation
+
+Mitiq's documentation is hosted at [mitiq.readthedocs.io](https://mitiq.readthedocs.io). A PDF version of the latest 
+release can be found [here](https://mitiq.readthedocs.io/_/downloads/en/latest/pdf/).
 
-## Contributing
-You can find information on contributing to `mitiq` code in the [contributing guidelines](CONTRIBUTING.md).
+## Developer information
 
-To contribute to the documentation, read the
-[instructions](docs/README-docs.md) in the `mitiq/docs` folder.
+We welcome contributions to Mitiq including bug fixes, feature requests, etc. Please see the 
+[contribution guidelines](CONTRIBUTING.md) for more details. To contribute to the documentation, please see these
+[documentation guidelines](docs/README-docs.md).
```

Comment:
```suggestion
[documentation guidelines](CONTRIBUTING_DOCS).
```

Reply:
Used the full url from Andrea's suggestion below.